pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -188,6 +190,13 @@ def load_data(self):
             )
         )
 
+    def load_uploads(self):
+        """
+        Copies upload fixtures into MEDIA_ROOT
+        """
+        print 'Copying fixtures from uploads directory to MEDIA_ROOT'
+        sh('cp -rf common/test/uploads/* test_root/uploads/')
```

Comment:
@benpatterson I have avoided the copying by just putting the fixture directly in `test_root/uploads/`, as per your suggestion.